crates/bevy_window/src/event.rs

```diff
@@ -330,3 +330,22 @@ pub struct WindowThemeChanged {
     /// The new system theme.
     pub theme: WindowTheme,
 }
+
+/// Application lifetime events
+#[derive(Event, Debug, Clone, PartialEq, Eq, Reflect)]
```

Comment:
Should this also be `Copy`?

```suggestion
#[derive(Event, Debug, Clone, Copy, PartialEq, Eq, Reflect)]
```